internal/kubernates/kubernate.go

```diff
@@ -0,0 +1,49 @@
+package kubernates
```

Comment:
is it purposely misspelled so that you could use a different directory from the "kubernetes" directory due to conflicts?

Reply:
(also curious about the file name)

Reply:
yeah, will update it in the complete PR.